lib/ParseSIL/ParseSIL.cpp

```diff
@@ -2071,6 +2071,31 @@ static bool parseAssignOwnershipQualifier(AssignOwnershipQualifier &Result,
   return false;
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+// Parse an index set, prefaced with the given label. Returns true on error.
+static bool parseIndexSet(Parser &P, StringRef label,
```

Comment:
Nothing in this function ensures what's parsed is a set. I'd suggest calling it "index list" instead.
```suggestion
static bool parseIndexList(Parser &P, StringRef label,
```

Reply:
Good point! Done in https://github.com/apple/swift/pull/29015/commits/8fbccc82b53643a94656c57574b62027570acd5b.